src/Manifest.cc

```diff
@@ -420,20 +420,33 @@ validateDepName(const StringRef name) {
     throw PoacError("dependency name must end with an alphanumeric character");
   }
 
-  // Only alphanumeric characters, `-`, `_`, and `/` are allowed.
```

Comment:
This pass should only check if this string only contains alphanumeric chars, `-`, `_`, `/`, and `.`.